docs/sources/migration-guide/migrating-from-thanos-or-prometheus.md

```diff
@@ -47,53 +47,33 @@ aws s3 cp <DIRECTORY> s3://<TENANT>/<DIRECTORY>
 gsutil -m cp -r <DIRECTORY> gs://<TENANT>/<DIRECTORY>
 ```
 
-## Migrating the block `meta.json` metadata using `metaconvert`
+## Block metadata
 
 Every block has a `meta.json` metadata file used by Grafana Mimir, Prometheus, and Thanos to identify the block contents.
 Each project has its own metadata conventions.
-The `metaconvert` tool migrates the `meta.json` metadata from one project to another.
 
-### Downloading `metaconvert`
+Ingesters from Mimir 2.1 and earlier added an external label into `meta.json` file to identify the tenant that owns the block.
```

Comment:
```suggestion
From the Grafana Mimir 2.1 release or earlier, the ingesters added an external label into the `meta.json` file to identify the tenant that owns the block.
```

Reply:
This information sounds like information that ought to be in the release notes rather than the core documentation.

Reply:
It is mentioned in the Mimir 2.2 release notes as:

```
* [CHANGE] Blocks uploaded by ingester no longer contain `__org_id__` label. Compactor now ignores this label and will compact blocks with and without this label together. `mimirconvert` tool will remove the label from blocks as "unknown" label. #1972
```